workflow-runtime/src/main/java/com/squareup/workflow1/internal/WorkflowNode.kt

```diff
@@ -225,10 +225,10 @@ internal class WorkflowNode<PropsT, StateT, OutputT, RenderingT>(
 
   private fun createSideEffectNode(
     key: String,
-    sideEffect: suspend () -> Unit
+    sideEffect: suspend CoroutineScope.() -> Unit
   ): SideEffectNode {
     val scope = this + CoroutineName("sideEffect[$key] for $id")
-    val job = scope.launch(start = LAZY) { sideEffect() }
+    val job = scope.launch(start = LAZY, block = sideEffect)
```

Comment:
Wow, is this the spot?

Reply:
Yep. This change wasn't actually necessary, since _this_ block already has the receiver, but this way we're allocating one fewer lambda.